adagrams/game.py

```diff
@@ -1,11 +1,154 @@
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}]
 def draw_letters():
-    pass
+    letters = []
+    remaining_letters_pool = {}
+    
+    if len(letters) < 11:
+    #getting the remaining amount of letters in the letter pool
+        for dict in LETTER_POOL:
+            for key, value in dict.items():
+                if key not in remaining_letters_pool and value > 0:
+                 remaining_letters_pool[key] = value
+    #adding random letter to the length of the letter list if it's less than ten
+    while len(letters) < 10:
+        random_letter = random.choice(list(remaining_letters_pool))
+    ##Removing the used letter from the remaining letter pool  
+        for letter in list(remaining_letters_pool):
+            if remaining_letters_pool[letter] == 0:
+                remaining_letters_pool.pop(letter)
+            elif random_letter == letter:
+                remaining_letters_pool[letter]-= 1
+                letters.append(random_letter)            
+    return letters
+
 
 def uses_available_letters(word, letter_bank):
-    pass
+    dict_word={}
+    word=word.upper()
+    word_list=[]
+    values_dict=[]
+    #Creating a list of each letter that is a string
+    for letter in word:
+        word_list.append(letter)
+    #Assigning a boolean to each letter if it appears in the list
+    for letter in word_list:
+        if letter in letter_bank:
+            freq_letter=word_list.count(letter)
+            freq_letter_bank=letter_bank.count(letter)
+            if freq_letter == freq_letter_bank:
+                dict_word[letter]=True
+            elif freq_letter>freq_letter_bank:
+                dict_word[letter]=False
+            elif freq_letter<freq_letter_bank:
+                dict_word[letter]=True
+        else:
+            dict_word[letter]=False
+    #Adding all of the booleans values for the given word to a list
+    for key,value in dict_word.items():
+        values_dict.append(value)
+    if False in values_dict:
+    #if there are any False values in the dict then return False
+        return False
+    else:
+        return True
+
 
 def score_word(word):
-    pass
+    #creating a new dictionary assinging values to each letter
+    letter_value={'A': 1, 
+    'B': 3, 
+    'C': 3, 
+    'D': 2, 
+    'E': 1, 
+    'F': 4, 
+    'G': 2, 
+    'H': 4, 
+    'I': 1, 
+    'J': 8, 
+    'K': 5, 
+    'L': 1, 
+    'M': 3, 
+    'N': 1, 
+    'O': 1, 
+    'P': 3, 
+    'Q': 10, 
+    'R': 1, 
+    'S': 1, 
+    'T': 1, 
+    'U': 1, 
+    'V': 4, 
+    'W': 4, 
+    'X': 8, 
+    'Y': 4, 
+    'Z': 10,}
+
+    total_sum=0
+    #Test case sensitivity
+    cap_word=word.upper()
+    if len(word) > 6:
+        total_sum=8
+    for letter in cap_word:
+        total_sum+=letter_value[letter]
+    return total_sum
```

Comment:
This looks great! My only suggestion would be to make the letter_value dictionary a constant outside of these functions.